src/hypothesis/searchstrategy/regex.py

```diff
@@ -294,7 +294,7 @@ def regex_strategy(regex):
     return maybe_pad(regex, base, left_pad, right_pad)
 
 
-def _strategy(codes, context, pattern):
+def _strategy(codes, context, is_text):
```

Comment:
In `regex_strategy` a bit above the `is_unicode` term is used for the same case. May be name it the same too for consistency?